lib/plucky/query.rb

```diff
@@ -64,6 +64,17 @@ def paginate(opts={})
         Pagination::Collection.new(docs, paginator)
       end
 
+      def find_in_batches(opts={})
+        batch_size = opts[:batch_size] || 1000
+        if block_given?
+          find_each(opts).each_slice(batch_size) do |documents|
```

Comment:
When not given a `:batch_size`, `find_each` does not use `1000`, but `each_slice` uses `1000`. Is this an intended behavior?